src/webui/components/relation/elementary/EditButton.tsx

```diff
@@ -0,0 +1,20 @@
+
+import { Button } from '^/webui/designsystem/module';
+
+export type Props = {
+    editHandler: () => void;
+};
+
+export default function Component({ editHandler }: Props)
```

Comment:
Why is this button here? And what's the difference with the `EditButton` from the creator module?